[2024-05-30T12:10:46.302+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-30T12:10:46.325+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stored_procedures.load_results manual__2024-05-30T12:10:34.497540+00:00 [queued]>
[2024-05-30T12:10:46.338+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stored_procedures.load_results manual__2024-05-30T12:10:34.497540+00:00 [queued]>
[2024-05-30T12:10:46.340+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-05-30T12:10:46.357+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_results> on 2024-05-30 12:10:34.497540+00:00
[2024-05-30T12:10:46.367+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1014) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-30T12:10:46.369+0000] {standard_task_runner.py:63} INFO - Started process 1016 to run task
[2024-05-30T12:10:46.368+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'stored_procedures', 'load_results', 'manual__2024-05-30T12:10:34.497540+00:00', '--job-id', '335', '--raw', '--subdir', 'DAGS_FOLDER/dag_example.py', '--cfg-path', '/tmp/tmply0sjz8y']
[2024-05-30T12:10:46.371+0000] {standard_task_runner.py:91} INFO - Job 335: Subtask load_results
[2024-05-30T12:10:46.424+0000] {task_command.py:426} INFO - Running <TaskInstance: stored_procedures.load_results manual__2024-05-30T12:10:34.497540+00:00 [running]> on host 29c09f03cc50
[2024-05-30T12:10:46.513+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Yamin' AIRFLOW_CTX_DAG_ID='stored_procedures' AIRFLOW_CTX_TASK_ID='load_results' AIRFLOW_CTX_EXECUTION_DATE='2024-05-30T12:10:34.497540+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-30T12:10:34.497540+00:00'
[2024-05-30T12:10:46.515+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-30T12:10:46.952+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-30T12:10:46.954+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-30T12:10:46.965+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=stored_procedures, task_id=load_results, execution_date=20240530T121034, start_date=20240530T121046, end_date=20240530T121046
[2024-05-30T12:10:46.986+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-30T12:10:47.013+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-30T12:10:47.016+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
